src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -211,4 +234,21 @@ export class ContentTypeFieldsDropZoneComponent extends BaseComponent implements
 
         return fields;
     }
+
+    removeField(fieldToDelete: Field): void {
```

Comment:
We need a method just to emit?

Reply:
I think it is better that tigger the event directly from the template